src/sentry/static/sentry/app/components/alert.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, {css} from 'react-emotion';
+import TextBlock from '../views/settings/components/text/textBlock';
+import InlineSvg from './inlineSvg';
+
+const AlertWrapper = styled.div`
+  margin: 0 0 ${p => p.theme.grid * 3}px;
```

Comment:
Yeah that seems right. I've usually done stuff like `small-space`, `x-small-space`, `xx-small-space` but since we are already in js land, a function is probably the right move. 

My only gripe is that I'm not a huge fan of `grid` as a catch all term, since this is more of a spacing system, but I'll save the naming stuff for later.